examples/vuejs3/src/rollbar.js

```diff
@@ -7,11 +7,23 @@ const rollbar = new Rollbar(config);
 export default {
   install(app) {
     app.config.errorHandler = (error, vm, info) => {
-      rollbar.error(error, { vueComponent: vm, info });
-      if (app.config.devtools) {
-        console.error(error);
-      }
-    };
+      
+      // In case the error is from the router or am helper
+      // calling vm could generate a loop and freeze the browser
+      // rollbar.error(error, { vueComponent: vm, info });
+      
+      rollbar.error(error, { info });
+      
+      if (app.config.devtools) console.error(error);
+    }
+
+    
+    app.config.warningHandler = (error, vm, info) => {
+      rollbar.warning(error, { info });
```

Comment:
I think a more appropriate example is to grab the most relevant and common data off the component, with coalesced fallbacks to empty data.

I also think it's best to stick with the existing format of multiline conditionals. It's a style choice, but the more common one, and since that's how the example was previously styled, I think we should stick to it for consistency across the docs
```suggestion
      // In case the error is from the router or a helper
      // calling vm could generate a loop and freeze the browser
      // Instead, we pick the relevant data off the vm instance
      
       const vueComponent = {
              name: vm.$options?.name ?? 'AnonymousComponent',
              props: vm.$props ?? {},
              data: vm.$data ?? {},
              attributes: vm.$attributes ?? {},
              route: vm.$route ?? {},
      };
      
      rollbar.error(error, { vueComponent, info });
      
       if (app.config.devtools) {
              console.error(error);
      }

    
    app.config.warningHandler = (error, vm, info) => {
      rollbar.warning(error, { vueComponent, info });
```